atm.js

```diff
@@ -1,21 +1,96 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var atm = {
+  currentInput: 0,
+  checkingBalance: 0,
+  savingsBalance: 0,
+  overage: 0,
+  buttonInitialize: function() {
```

Comment:
I love that you placed all of these click events into one method, great job :+1: